app/src/main/java/co/touchlab/droidconandroid/shared/interactors/RsvpInteractor.java

```diff
@@ -2,26 +2,33 @@
 
 import android.support.annotation.NonNull;
 
+import com.birbit.android.jobqueue.JobManager;
+
 import java.util.UUID;
 
+import co.touchlab.droidconandroid.DroidconApplication;
 import co.touchlab.droidconandroid.shared.data.DatabaseHelper;
 import co.touchlab.droidconandroid.shared.data.Event;
+import co.touchlab.droidconandroid.shared.tasks.persisted.RsvpJob;
 import co.touchlab.squeaky.dao.Dao;
 import io.reactivex.Single;
 
 /**
  * Created by kgalligan on 4/7/16.
  */
 public class RsvpInteractor {
-    private final Long eventId;
+    private final Long           eventId;
     private final DatabaseHelper helper;
+    private final JobManager     jobManager;
 
     public RsvpInteractor(DatabaseHelper helper, Long eventId) {
         this.eventId = eventId;
         this.helper = helper;
+        this.jobManager = DroidconApplication.getInstance().getJobManager();
     }
 
     public Single<Event> addRsvp() {
+        jobManager.addJobInBackground(new RsvpJob(eventId));
```

Comment:
I think it makes sense for call to be part of the stream below and call it in a doOnNext after we have updated the event in the db. This is more aligned with the way it used to work with tasks and like this it triggers the job even the person calling addRsvp never subscribes